src/index.ts

```diff
@@ -15,10 +15,7 @@ import { pack } from './pack';
 import { preOffline } from './pre-offline';
 import { preLocal } from './pre-local';
 import { trimExtension } from './utils';
-
-export const SERVERLESS_FOLDER = '.serverless';
```

Comment:
https://github.com/floydspace/serverless-esbuild/blob/fb8345d3f596fad4eb0452780e1cb4572cc09b9f/src/index.ts#L441

Because of this line for some reason when the `pack.test.ts` file ran these variables were coming as `undefined`. Moved to their own file.

Reply:
Alternatively we could declare them on the class itself. I'm open to suggestions.

Reply:
it would be good to have `constants.ts` to keep all the static values there. 
`folders` is too narrow, imo